apps/website-25/src/pages/api/courses/exercises/[exerciseId]/response.ts

```diff
@@ -0,0 +1,88 @@
+import { z } from 'zod';
+import createHttpError from 'http-errors';
+import db from '../../../../../lib/api/db';
+import { makeApiRoute } from '../../../../../lib/api/makeApiRoute';
+import {
+  ExerciseResponse,
+  exerciseResponseTable,
+} from '../../../../../lib/api/db/tables';
+
+export type GetExerciseResponseResponse = {
```

Comment:
😄

(but it makes sense and is in line with convention of others, I think you should keep it this way)